frontend/src/components/homepage/Leagues.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useEffect, useState } from 'react';
+import { backdropClasses, Box, Button, Stack, styled, TextField, Typography } from '@mui/material';
+
+const LeaguesContainer = styled(Box)({
+    height: "fit-content",
+    padding: "15px",
+    display: "flex", 
+    alignItems: "center",
+    flexDirection: "column",
+    gap: "10px",
+    border: "2px solid #D9D9D9", 
+    boxShadow: "2px 5px #D9D9D9",
+    borderRadius: "10px",
+});
+
+const Title = styled(Box)({
+    fontWeight: "bold",
+    fontSize: "24px",
+    display: "flex",
+    alignSelf: "flex-start",
+});
+
+const CustomButton = styled(Button)({
+    backgroundColor: " #E5B226",
+    color: "#FBF9F9",
+    boxShadow: "0px 2px #D9D9D9",
+    borderRadius: "10px",
+    fontWeight: "bold",
+    "&:hover":{
+        backgroundColor: "rgb(140, 109, 23)"
+    }
+});
+
+const Content = styled(Box)({
+    height:"fit-content",
+    width: "100%",
+    paddingBottom: "15px",
+    display: "flex",
+    gap: "50px",
+    justifyContent: "space-around",
+    alignItems: "center"
+});
+
+const LeftContent = styled(Box)({
+    height: "fit-content",
+    display: "flex",
+    flexDirection: "column",
+    gap: "5px"
+});
+
+const LeagueButton = styled(Box)((props:{is_active: number}) => ({
+    fontWeight: props.is_active ? "bold" : "none",
+    color: props.is_active ? "#158030" : "#000000",
+    cursor: "pointer",
+    "&:hover": {
+        fontWeight : "bold"
+    }
+}));
+
+const RightContent = styled(Box)({
```

Comment:
comme pour leftContent, ici le rightContent peut etre précisé
```suggestion
const ButtonsContainer= styled(Box)({
```